jenkins-scripts/gazebo-one_liner-homebrew.bash

```diff
@@ -9,6 +9,8 @@ echo '# BEGIN SECTION: cleanup brew installation'
 echo '# END SECTION'
 
 echo '# BEGIN SECTION: run the one-liner installation'
-# TODO: change this to 'curl -ssL https://get.gazebosim.org' once that supports https
-curl -ssL https://github.com/gazebo-tooling/release-tools/raw/master/one-line-installations/gazebo.sh | sh -x
+curl -ssL https://get.gazebosim.org | sh -x
```

Comment:
```suggestion
curl -sSL https://get.gazebosim.org | sh -x
```
One uppercase S to show errors?

Reply:
applied in https://github.com/gazebo-tooling/release-tools/pull/1064/commits/0abf03efc5978aecd6fec5b9215b7d0e253e4f4e